rusk/src/lib/builder/node.rs

```diff
@@ -297,6 +297,24 @@ impl RuskNodeBuilder {
         }
 
         node.inner().initialize(&mut service_list).await?;
+
+        #[cfg(feature = "archive")]
+        {
```

Comment:
Would be possible to run this only if the current stored accounts is empty? This way we only performed the migration once